ksql-rest-model/src/main/java/io/confluent/ksql/rest/Errors.java

```diff
@@ -190,4 +189,17 @@ public static Response serverNotReady(final KsqlErrorMessage error) {
         .entity(error)
         .build();
   }
+  
+  
+  public Errors(final ErrorMessages errorMessages) {
+    this.errorMessages = errorMessages;
+  }
+
+  public Response accessDeniedFromKafka(final Exception e) {
+    return constructAccessDeniedFromKafkaResponse(errorMessages.kafkaAuthorizationErrorMessage(e));
+  }
+
+  public String webSocketKafkaAuthorizationErrorMessage(final Exception e) {
```

Comment:
nit: I don't think this necessarily needs `webSocket` in the name.  It's just returning the `kafkaAuthorizationErrorMessage`, so just call it `kafkaAuthorizationErrorMessage`.  The fact its _currently_ only called from the websocket impl is kind of irrelevant.